app/views/sephora_style_guide/home/_icons.slim

```diff
@@ -443,6 +443,148 @@ ruby:
       "size": "icon_30x20"
     }
   ]
+  countries = [
+    {
+      "name": "Hong Kong",
+      "class": "hong-kong"
+    },
+    {
+      "name": "Singapore",
+      "class": "singapore"
+    },
+    {
+      "name": "Philippines",
+      "class": "philippines"
+    },
+    {
+      "name": "Malaysia",
+      "class": "malaysia"
+    },
+    {
+      "name": "Australia",
+      "class": "australia"
+    },
+    {
+      "name": "New Zealand",
+      "class": "new-zealand"
+    },
+    {
+      "name": "Thailand",
+      "class": "thailand"
+    },
+    {
+      "name": "Indonesia",
+      "class": "indonesia"
+    },
+    {
+      "name": "Canada",
+      "class": "canada"
+    },
+    {
+      "name": "Mexico",
+      "class": "mexico"
+    },
+    {
+      "name": "United States",
+      "class": "united-states"
+    },
+    {
+      "name": "Brazil",
+      "class": "brazil"
+    },
+    {
+      "name": "Czech Republic",
+      "class": "czech-republic"
+    },
+    {
+      "name": "Denmark",
+      "class": "denmark"
+    },
+    {
+      "name": "France",
+      "class": "france"
+    },
+    {
+      "name": "Germany",
+      "class": "germany"
+    },
+    {
+      "name": "Greece",
+      "class": "greece"
+    },
+    {
+      "name": "Italy",
+      "class": "italy"
+    },
+    {
+      "name": "Luxembourg",
+      "class": "luxembourg"
+    },
+    {
+      "name": "Monaco",
+      "class": "monaco"
+    },
+    {
+      "name": "Poland",
+      "class": "poland"
+    },
+    {
+      "name": "Portugal",
+      "class": "portugal"
+    },
+    {
+      "name": "Romania",
+      "class": "romania"
+    },
+    {
+      "name": "Russia",
+      "class": "russia"
+    },
+    {
+      "name": "Serbia",
+      "class": "serbia"
+    },
+    {
+      "name": "Slovakia",
+      "class": "slovakia"
+    },
+    {
+      "name": "Spain",
+      "class": "spain"
+    },
+    {
+      "name": "Switzerland",
+      "class": "switzerland"
+    },
+    {
+      "name": "Turkey",
+      "class": "turkey"
+    },
+    {
+      "name": "Bahrain",
+      "class": "bahrain"
+    },
+    {
+      "name": "China",
+      "class": "china"
+    },
+    {
+      "name": "Kuwait",
+      "class": "kuwait"
+    },
+    {
+      "name": "Qatar",
+      "class": "qatar"
+    },
+    {
+      "name": "Saudi Arabia",
+      "class": "saudi-arabia"
+    },
+    {
+      "name": "United Arab Emirate",
```

Comment:
I am just being picky here but the name is actually `United Arab Emirates` with "s"